iroh-relay/src/client.rs

```diff
@@ -1,3 +1,5 @@
+//! Exposes [`Client`], which allows to establish connections to a relay server.
+//!
 //! Based on tailscale/derp/derphttp/derphttp_client.go
```

Comment:
Sorry, can we make this at least a non-documentation comment? :sweat_smile:

Reply:
I would rather not remove it but I get why people would prefer to not have it as crate docs.. in order to make this merge ready I did the change